packages/suite/src/support/messages.ts

```diff
@@ -3356,7 +3356,7 @@ export default defineMessages({
         id: 'TR_TESTNET_COINS_LABEL',
     },
     TR_THE_PIN_LAYOUT_IS_DISPLAYED: {
-        defaultMessage: 'Check your Trezor screen for the keypad layout.',
+        defaultMessage: 'Check your <b>{deviceLabel}</b> screen for the keypad layout.',
```

Comment:
it's not an HTML, I think <b>  would not work out of the box, you have to add something like:
`b: text => <b>{text}</b>` to values for TR_THE_PIN_LAYOUT_IS_DISPLAYED Translation component next to deviceLabel...

Reply:
Thanks, I was looking to other translations that use <b> but just how they're handled in the `messages.ts` file.